src/js/utils/spatial-navigation-keycode.js

```diff
@@ -0,0 +1,49 @@
+// /**
+//  * @file spatial-navigation-keycode.js
+//  */
+
+import * as browser from './browser.js';
+
+// Determine the keycode for the 'back' key based on the platform
+const backKeyCode = browser.IS_TIZEN ? 10009 : browser.IS_WEBOS ? 461 : 8;
+
+const SpatialNavKeycode = {
+  keyCodes: {
+    codes: {
+      play: 415,
+      pause: 19,
+      ff: 417,
+      rw: 412,
+      back: backKeyCode
+    },
+    names: {
+      415: 'play',
+      19: 'pause',
+      417: 'ff',
+      412: 'rw',
+      backKeyCode: 'back'
```

Comment:
This will literally create a key called "backKeyCode" instead of using the value of backKeyCode as the key. You need to use dynamic property keys to fix this.

Reply:
Fixed in last commit